dist/webinstaller/quick-lint-js-template.json

```diff
@@ -7,7 +7,7 @@
       "channel": "stable",
       "date": "${releaseDate}",
       "os": "linux",
-      "arch": "",
+      "arch": "x86_64/amd64",
```

Comment:
Must fix: This doesn't look right. I would expect either "x86_64" or "amd64", not both. Did you test this? What does webinstaller expect? What do other webinstaller packages use?

Reply:
You are right. We can't really do that, I will create an object for each of the 3 architectures for the 3 OS's.

Reply:
> You are right. We can't really do that, I will create an object for each of the 3 architectures for the 3 OS's.

@strager  does this seems like the right thing to do ?